### ISSUE TITLE
Poisson Distribution Module not found

### ISSUE THREAD

**Issue by edarca** [awslabs/gluon-ts#1327]
Hi! thank you so much for this amazing tool!

Some days ago I upgrade my gluonts version. By the moment the predictions performed well, but once I try to use Poisson distribution, an error appeared. 

```from gluonts.distribution.neg_binomial import NegativeBinomialOutput
from gluonts.distribution.multivariate_gaussian import MultivariateGaussianOutput
from gluonts.distribution.poisson import PoissonOutput


estimator1 = DeepAREstimator(
    prediction_length=prediction_length,
    freq="W",
    num_layers=2, 
    num_cells=40,
    distr_output = NegativeBinomialOutput(),
    use_feat_static_cat=True,
    cardinality=cardinalities, 
    #dropout_rate=0.1,
    trainer=Trainer(ctx="cpu",
        learning_rate=1e-3,
        epochs=30,
        num_batches_per_epoch=600,             
        batch_size=32                          
    )
)

predictor = estimator1.train(dataset_train)
```

Error:
```
ModuleNotFoundError: No module named 'gluonts.distribution.poisson'
```

With another version of gluonts, I remember that this code worked correctly.

## Environment
- Operating system: Windows 10
- Python version: 3.6
- GluonTS version: 0.4.3
- MXNet version: 1.7.0.post2

**Comment by lostella** [comment]
@edarca hi! the `Poisson` and `PoissonOutput` classes were added in v0.5.0, so they're not there in 0.4.3. Can you try upgrading the package? Also, from v0.6.0 onwards distributions should be imported `from gluonts.mx.distribution` (note the additional `mx`).

**Comment by edarca** [comment]
Thanks @lostella! Solved with 0.6.6 and using module gluonts.mx
